src/Aspire.Hosting/Dcp/DcpOptions.cs

```diff
@@ -113,16 +111,22 @@ public void Configure(DcpOptions options)
         var dcpPublisherConfiguration = configuration.GetSection(DcpPublisher);
         var assemblyMetadata = appOptions.Assembly?.GetCustomAttributes<AssemblyMetadataAttribute>();
 
-        if (!string.IsNullOrEmpty(dcpPublisherConfiguration[nameof(options.CliPath)]))
+        // Find dcp tools path in the following order:
+        // 1. Environment variable DOTNET_ASPIRE_DCP_DIR
+        // 2. Configuration value
```

Comment:
Environment variables already factor into the Configuration. I don't think we need a separate/new environment variable. Instead, the tests can just set the `DcpPublisher__CliPath` environment variable.

Reply:
We could also compute all 3 paths based on this `CliPath` then. I'm not sure why that is not being done currently.